storage/libradb/src/ledger_store/mod.rs

```diff
@@ -41,6 +41,8 @@ pub(crate) struct LedgerStore {
     latest_ledger_info: ArcSwap<Option<LedgerInfoWithSignatures>>,
 }
 
+const MAX_NUM_EPOCH_CHANGE_LEDGER_INFO: usize = 100;
```

Comment:
I think this might not make sense, at least right now. If a client has a really old `client_known_version` such that there are more than `MAX_NUM_EPOCH_CHANGE_LEDGER_INFO` epoch changes between `client_known_version` and the latest version, the implementation below will always return an error and there's no way for the client to catch up. Do I understand this correctly?

Reply:
Yes we need to implement a protocol to hint the client to loop or boot from a waypoint. This protects the server only.

Reply:
I'd just document this a little bit in the code so anyone who reads the code is able to understand.